### PR TITLE
Pioneer 18 Leg 1 Telemtered

### PR DESCRIPTION
**Pull request by cdobs**
Creates telemetered ingestion csvs for platforms deployed on Pioneer 18 Leg 1

### CODE REVIEW

File: CP05MOAS-GL339/CP05MOAS-GL339_D00015_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/D00015/merged-from-glider/cp_*.mrg,CP05MOAS-GL339-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/D00015/merged-from-glider/cp_*.mrg,CP05MOAS-GL339-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/D00015/merged-from-glider/cp_*.mrg,CP05MOAS-GL339-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/D00015/merged-from-glider/cp_*.mrg,CP05MOAS-GL339-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/D00015/merged-from-glider/cp_*.mrg,CP05MOAS-GL339-05-PARADM000,telemetered,Available,
```
Comment (snwhite) [inline CP05MOAS-GL339/CP05MOAS-GL339_D00015_ingest.csv:6]: Looks good.

File: CP05MOAS-GL380/CP05MOAS-GL380_D00012_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/D00012/merged-from-glider/cp_*.mrg,CP05MOAS-GL380-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/D00012/merged-from-glider/cp_*.mrg,CP05MOAS-GL380-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/D00012/merged-from-glider/cp_*.mrg,CP05MOAS-GL380-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/D00012/merged-from-glider/cp_*.mrg,CP05MOAS-GL380-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/D00012/merged-from-glider/cp_*.mrg,CP05MOAS-GL380-05-PARADM000,telemetered,Available,
```
Comment (snwhite) [inline CP05MOAS-GL380/CP05MOAS-GL380_D00012_ingest.csv:6]: Looks good.

File: CP05MOAS-GL388/CP05MOAS-GL388_D00013_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/D00013/merged-from-glider/cp_*.mrg,CP05MOAS-GL388-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/D00013/merged-from-glider/cp_*.mrg,CP05MOAS-GL388-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/D00013/merged-from-glider/cp_*.mrg,CP05MOAS-GL388-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/D00013/merged-from-glider/cp_*.mrg,CP05MOAS-GL388-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/D00013/merged-from-glider/cp_*.mrg,CP05MOAS-GL388-05-PARADM000,telemetered,Available,
```
Comment (snwhite) [inline CP05MOAS-GL388/CP05MOAS-GL388_D00013_ingest.csv:6]: Looks good.

File: CP05MOAS-GL388/CP05MOAS-GL388_D00014_ingest.csv [hunk 4]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/D00014/merged-from-glider/cp_*.mrg,CP05MOAS-GL388-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/D00014/merged-from-glider/cp_*.mrg,CP05MOAS-GL388-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/D00014/merged-from-glider/cp_*.mrg,CP05MOAS-GL388-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/D00014/merged-from-glider/cp_*.mrg,CP05MOAS-GL388-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/D00014/merged-from-glider/cp_*.mrg,CP05MOAS-GL388-05-PARADM000,telemetered,Available,
```
Comment (snwhite) [inline CP05MOAS-GL388/CP05MOAS-GL388_D00014_ingest.csv:6]: Looks good.

File: CP04OSSM/CP04OSSM_D00015_ingest.csv [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00015/cg_data/dcl27/syslog/*.syslog.log,CP04OSSM-RID27-00-DCLENG000,telemetered,Available,
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00015/cg_data/dcl27/optaa1_decimated/*.optaa1.log,CP04OSSM-RID27-01-OPTAAD000,telemetered,Available,
+mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00015/cg_data/dcl27/flort/*.flort.log,CP04OSSM-RID27-02-FLORTD000,telemetered,Available,
+mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00015/cg_data/dcl27/ctdbp1/*.ctdbp1.log,CP04OSSM-RID27-03-CTDBPC000,telemetered,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00015/cg_data/dcl27/dosta1/*.dosta1.log,CP04OSSM-RID27-04-DOSTAD000,telemetered,Available,
+mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00015/irid2shore/cpm_status*.txt,CP04OSSM-SBC11-00-CPMENG000,telemetered,Available,
```
Comment (snwhite) [inline CP04OSSM/CP04OSSM_D00015_ingest.csv:25]: All looks good, but there is no irid2shore folder in the Raw Data Repo.

File: CP03ISSM/CP03ISSM_D00015_ingest.csv [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP03ISSM/D00015/cg_data/dcl27/syslog/*.syslog.log,CP03ISSM-RID27-00-DCLENG000,telemetered,Available,
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP03ISSM/D00015/cg_data/dcl27/optaa1_decimated/*.optaa1.log,CP03ISSM-RID27-01-OPTAAD000,telemetered,Available,
+mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP03ISSM/D00015/cg_data/dcl27/flort/*.flort.log,CP03ISSM-RID27-02-FLORTD000,telemetered,Available,
+mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CP03ISSM/D00015/cg_data/dcl27/ctdbp1/*.ctdbp1.log,CP03ISSM-RID27-03-CTDBPC000,telemetered,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_telemetered_driver,/omc_data/whoi/OMC/CP03ISSM/D00015/cg_data/dcl27/dosta1/*.dosta1.log,CP03ISSM-RID27-04-DOSTAD000,telemetered,Available,
+mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_telemetered_driver,/omc_data/whoi/OMC/CP03ISSM/D00015/irid2shore/cpm_status*.txt,CP03ISSM-SBC11-00-CPMENG000,telemetered,Available,
```
Comment (snwhite) [inline CP03ISSM/CP03ISSM_D00015_ingest.csv:25]: All looks good, but there is no irid2shore folder in the Raw Data Repo.

File: CP01CNSM/CP01CNSM_D00016_ingest.csv [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00016/cg_data/dcl27/syslog/*.syslog.log,CP01CNSM-RID27-00-DCLENG000,telemetered,Available,
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00016/cg_data/dcl27/optaa1_decimated/*.optaa1.log,CP01CNSM-RID27-01-OPTAAD000,telemetered,Available,
+mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00016/cg_data/dcl27/flort/*.flort.log,CP01CNSM-RID27-02-FLORTD000,telemetered,Available,
+mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00016/cg_data/dcl27/ctdbp1/*.ctdbp1.log,CP01CNSM-RID27-03-CTDBPC000,telemetered,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00016/cg_data/dcl27/dosta1/*.dosta1.log,CP01CNSM-RID27-04-DOSTAD000,telemetered,Available,
+mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00016/irid2shore/cpm_status*.txt,CP01CNSM-SBC11-00-CPMENG000,telemetered,Available,
```
Comment (snwhite) [inline CP01CNSM/CP01CNSM_D00016_ingest.csv:25]: All looks good, but there is no irid2shore folder in the Raw Data Repo.

**Review by snwhite — APPROVED** [review]

File: CP01CNSM/CP01CNSM_D00016_ingest.csv [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00016/cg_data/dcl12/syslog/*.syslog.log,CP01CNSM-SBD12-00-DCLENG000,telemetered,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00016/cg_data/dcl12/hyd2/*.hyd2.log,CP01CNSM-SBD12-03-HYDGN0000,telemetered,Available,
+mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00016/cg_data/dcl12/pco2a/*.pco2a.log,CP01CNSM-SBD12-04-PCO2AA000,telemetered,Available,
+mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00016/cg_data/dcl12/wavss/*.wavss.log,CP01CNSM-SBD12-05-WAVSSA000,telemetered,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00016/cg_data/dcl12/metbk2/*.metbk2.log,CP01CNSM-SBD12-06-METBKA000,telemetered,Available,
+mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00016/cg_data/dcl12/fdchp/*.fdchp.log,CP01CNSM-SBD12-08-FDCHPA000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP01CNSM/CP01CNSM_D00016_ingest.csv:35]: good

File: CP03ISSM/CP03ISSM_D00015_ingest.csv [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+#mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP03ISSM/D00015/cg_data/dcl11/mopak/*.mopak.log,CP03ISSM-SBD11-01-MOPAK0000,telemetered,Not Expected,Data will be available upon recovery
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP03ISSM/D00015/cg_data/dcl11/hyd1/*.hyd1.log,CP03ISSM-SBD11-02-HYDGN0000,telemetered,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP03ISSM/D00015/cg_data/dcl11/metbk/*.metbk.log,CP03ISSM-SBD11-06-METBKA000,telemetered,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP03ISSM/D00015/cg_data/dcl12/syslog/*.syslog.log,CP03ISSM-SBD12-00-DCLENG000,telemetered,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP03ISSM/D00015/cg_data/dcl12/hyd2/*.hyd2.log,CP03ISSM-SBD12-03-HYDGN0000,telemetered,Available,
+mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_telemetered_driver,/omc_data/whoi/OMC/CP03ISSM/D00015/cg_data/dcl12/pco2a/*.pco2a.log,CP03ISSM-SBD12-04-PCO2AA000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP03ISSM/CP03ISSM_D00015_ingest.csv:32]: good

File: CP04OSSM/CP04OSSM_D00015_ingest.csv [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+#mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00015/cg_data/dcl11/mopak/*.mopak.log,CP04OSSM-SBD11-01-MOPAK0000,telemetered,Not Expected,Data will be available upon recovery
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00015/cg_data/dcl11/hyd1/*.hyd1.log,CP04OSSM-SBD11-02-HYDGN0000,telemetered,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00015/cg_data/dcl11/metbk/*.metbk.log,CP04OSSM-SBD11-06-METBKA000,telemetered,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00015/cg_data/dcl12/syslog/*.syslog.log,CP04OSSM-SBD12-00-DCLENG000,telemetered,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00015/cg_data/dcl12/hyd2/*.hyd2.log,CP04OSSM-SBD12-03-HYDGN0000,telemetered,Available,
+mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00015/cg_data/dcl12/pco2a/*.pco2a.log,CP04OSSM-SBD12-04-PCO2AA000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP04OSSM/CP04OSSM_D00015_ingest.csv:32]: good

File: CP05MOAS-GL339/CP05MOAS-GL339_D00015_ingest.csv (hunk 1 above)
Comment (rtravis-whoi) [inline CP05MOAS-GL339/CP05MOAS-GL339_D00015_ingest.csv:6]: good

File: CP05MOAS-GL380/CP05MOAS-GL380_D00012_ingest.csv (hunk 2 above)
Comment (rtravis-whoi) [inline CP05MOAS-GL380/CP05MOAS-GL380_D00012_ingest.csv:6]: good

File: CP05MOAS-GL388/CP05MOAS-GL388_D00013_ingest.csv (hunk 3 above)
Comment (rtravis-whoi) [reply CP05MOAS-GL388/CP05MOAS-GL388_D00013_ingest.csv:6]: good

File: CP05MOAS-GL388/CP05MOAS-GL388_D00014_ingest.csv (hunk 4 above)
Comment (rtravis-whoi) [inline CP05MOAS-GL388/CP05MOAS-GL388_D00014_ingest.csv:6]: good

**Review by rtravis-whoi — APPROVED** [review]

**Pull request merged**